src/Files.App/Services/Storage/StorageNetworkService.cs

```diff
@@ -40,102 +36,180 @@ public ObservableCollection<ILocatableFolder> Shortcuts
 		/// </summary>
 		public NetworkService()
 		{
-			var networkItem = new DriveItem()
+			var item = new DriveItem()
 			{
 				DeviceID = "network-folder",
 				Text = "Network".GetLocalizedResource(),
 				Path = Constants.UserEnvironmentPaths.NetworkFolderPath,
 				Type = DriveType.Network,
 				ItemType = NavigationControlItemType.Drive,
+				MenuOptions = new ContextMenuOptions()
+				{
+					IsLocationItem = true,
+					ShowShellItems = true,
+					ShowProperties = true,
+				},
 			};
 
-			networkItem.MenuOptions = new ContextMenuOptions()
-			{
-				IsLocationItem = true,
-				ShowEjectDevice = networkItem.IsRemovable,
-				ShowShellItems = true,
-				ShowProperties = true,
-			};
+			item.MenuOptions.ShowEjectDevice = item.IsRemovable;
+
 			lock (_Computers)
-				_Computers.Add(networkItem);
+				_Computers.Add(item);
 		}
 
 		/// <inheritdoc/>
 		public async Task<IEnumerable<ILocatableFolder>> GetComputersAsync()
 		{
-			var result = await Win32Helper.GetShellFolderAsync(guid, false, true, 0, int.MaxValue);
+			return await Task.Run(GetComputers);
```

Comment:
What does "Computers" represent?

Reply:
@hishitetsu would have to answer this.

Reply:
Network Computers.
![image](https://github.com/user-attachments/assets/d392ee7a-8663-4d42-8f69-7913b4defb4f)